launcher/minecraft/launch/ExtractNatives.cpp

```diff
@@ -91,15 +88,16 @@ void ExtractNatives::executeTask()
         return;
     }
     auto settings = minecraftInstance->settings();
+
+    // We only need OpenAL here, as modern versions of LWJGL (3+) are handled by JVM args, while older versions (2) didn't have GLFW
     bool nativeOpenAL = settings->get("UseNativeOpenAL").toBool();
-    bool nativeGLFW = settings->get("UseNativeGLFW").toBool();
 
     auto outputPath  = minecraftInstance->getNativePath();
     auto javaVersion = minecraftInstance->getJavaVersion();
     bool jniHackEnabled = javaVersion.major() >= 8;
     for(const auto &source: toExtract)
     {
-        if(!unzipNatives(source, outputPath, jniHackEnabled, nativeOpenAL, nativeGLFW))
+        if(!unzipNatives(source, outputPath, jniHackEnabled, nativeOpenAL))
```

Comment:
This doesn't break the feature for versions that use lwjgl < 3?

Reply:
Did you read the comment a few lines up?

Reply:
... the horizontal scroll cut off the end of the comment and I wasn't smart enough to scroll right to see it.